source/fundamentals/connection/connection-options.txt

```diff
@@ -216,6 +216,11 @@ parameters of the connection URI to specify the behavior of the client.
      - ``false``
      - The ``ssl`` is an alias for the ``tls`` option.
 
+   * - **timeoutMS**
+     - non-negative integer
+     - unset (feature is not enabled)
+     - Specifies the time limit for the full execution of an operation.
```

Comment:
Suggestion: add the unit (milliseconds). For example, "serverSelectionTimeoutMS" uses the description "Specifies the timeout, in milliseconds, ..."

Reply:
also added to other options where applicable